### PR TITLE
deploy many changes and break some stuff

### PR DESCRIPTION
**Pull request by MaxAFriedrich**

### CODE REVIEW

File: components/Logo.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
-  <div style="display:flex;">
-    <img src="/img/logo/light.svg" alt="CompSoc Logo"/>
+  <div style="display: flex">
```
Comment (moddedTechnic) [inline components/Logo.vue:2]: shouldn't this be an uno class?

Comment (moddedTechnic) [reply components/Logo.vue:2]: the `flex` styling

File: components/events/Add.vue [hunk 2]
```diff
@@ -78,60 +88,133 @@ async function addEvent() {
...
+          type="text"
+          id="organizer"
+          v-model="formData.organizer"
+          required
+        />
       </div>
-      <span>All times should be entered in the time that they will run. Do not adujust for time zones.</span>
+      <span
+        >All times should be entered in the time that they will run. Do not
+        adujust for time zones.</span
```
Comment (moddedTechnic) [inline components/events/Add.vue:159]: @MaxAFriedrich spelling please

**Review by moddedTechnic — APPROVED** [review]

File: app.vue [hunk 3]
```diff
@@ -1,13 +1,17 @@
 <template>
   <div class="dark:bg-darkgrey dark:text-gray-300 bg-#e7e7e7">
-    <NuxtLayout name="default">
+    <NuxtLayout>
       <NuxtPage />
     </NuxtLayout>
+    <noscript>
```
Comment (hugohabicht01) [inline app.vue:6]: not working, but who cares

File: server/api/events/all.get.ts [hunk 4]
```diff
@@ -1,19 +1,38 @@
...
+interface OffsetData {
+  years?: string;
+  months?: string;
+  weeks?: string;
+  days?: string;
+}
+
 export default defineEventHandler(event => {
-  // Calculate the current date and a year in the future
+  const offsetData = getQuery<OffsetData>(event);
```
Comment (hugohabicht01) [inline server/api/events/all.get.ts:10]: once this is live, we should refactor this to use h3-zodi

**Review by hugohabicht01 — APPROVED** [review]
lgtm

**Pull request merged**